security_monkey/scheduler.py

```diff
@@ -141,6 +147,9 @@ def setup_scheduler():
             if auditors:
                 scheduler.add_cron_job(_audit_changes, hour=10, day_of_week="mon-fri", args=[account, auditors, True])
 
+                # Clear out old exceptions:
+                scheduler.add_cron_job(_clear_old_exceptions, hour=3, minute=0)
```

Comment:
It looks like you're going to schedule _clear_old_exceptions to run at 3am for every account that has an auditor.

Probably want to put this after the `for account in accounts` loop.